### PR TITLE
Update missing translation about cis、istio、monitoring and logging parts for v2.6

### PR DESCRIPTION
**Pull request by GGGitBoy**
Update part of Chinese missing docs for the 2.6 which includes:

- cis-scan
- istio
- monitoring
- logging
- experimental-features

### CODE REVIEW

**Comment by GGGitBoy** [comment]
@JacieChao Please help me review this PR. Thank you!

File: i18n/zh/docusaurus-plugin-content-docs/version-2.6/integrations-in-rancher/istio/istio.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+如需了解如何设置 Istio 并在项目中使用它，请参阅[设置指南](../../how-to-guides/advanced-user-guides/istio-setup-guide/istio-setup-guide.md)。
+
+## 卸载 Istio
+
+要从集群、命名空间或工作负载中删除 Istio 组件，请参阅[卸载 Istio](disable-istio.md)。
+
+
+
+。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.6/integrations-in-rancher/istio/istio.md:71]: 这里是多了一个标点？

File: i18n/zh/docusaurus-plugin-content-docs/version-2.6/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Monitoring 应用：
+
+- 监控集群节点、Kubernetes 组件和软件部署的状态和进程。
+- 根据 Prometheus 收集的指标定义告警。
+- 创建自定义 Grafana 仪表板。
+- 使用 Prometheus Alertmanager 通过电子邮件、Slack、PagerDuty 等配置告警通知。
+- 根据 Prometheus 收集的指标，将预先计算的、经常需要的，或计算成本高的表达式定义为新的时间序列。
+- 通过 Prometheus Adapter，将从 Prometheus 收集的指标公开给 Kubernetes Custom Metrics API，以便在 HPA 中使用。
+
+## How Monitoring Works
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.6/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md:29]: 这个标题没翻译

File: i18n/zh/docusaurus-plugin-content-docs/version-2.6/reference-guides/monitoring-v2-configuration/monitoring-v2-configuration.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Monitoring V2 配置
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.6/reference-guides/monitoring-v2-configuration/monitoring-v2-configuration.md:2]: title名称为 Monitoring Configuration Reference

Comment (GGGitBoy) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.6/reference-guides/monitoring-v2-configuration/monitoring-v2-configuration.md]: updated

File: i18n/zh/docusaurus-plugin-content-docs/version-2.6/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md (hunk 2 above)
Comment (GGGitBoy) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.6/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md]: updated

File: i18n/zh/docusaurus-plugin-content-docs/version-2.6/integrations-in-rancher/istio/istio.md (hunk 1 above)
Comment (GGGitBoy) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.6/integrations-in-rancher/istio/istio.md]: updated

**Review by btat — APPROVED** [review]

**Pull request merged**